packages/init/src/index.ts

```diff
@@ -15,11 +16,20 @@ const CONFIG_PREFIX = '--force';
  */
 
 export default function initializeInquirer(...args: string[]): Function | void {
-    const packages = args;
-    const includesDefaultPrefix = packages.includes(AUTO_PREFIX);
-    const generateConfig = packages.includes(CONFIG_PREFIX);
-    if (packages.length === 0 || includesDefaultPrefix || generateConfig) {
+    const cli = new WebpackCLI();
+    const { opts, unknownArgs } = cli.argParser(options, args, true);
+    const { auto: includesDefaultPrefix, force: generateConfig } = opts;
+
+    // Filter out cli-flags from unknownArgs
+    const unknownFlags = unknownArgs.filter((arg) => arg.startsWith('--'));
+
+    if (unknownFlags.length > 0) {
+        logger.warn(`Unknown ${unknownFlags.length === 1 ? 'argument' : 'arguments'}: ${unknownFlags}`);
```

Comment:
I think we should throw an error in this case and exit with code `-1`

Reply:
`serve` seems to follow a similar approach. Should we change it?

https://github.com/webpack/webpack-cli/blob/f1876c125be90e0b0ae96e6e9dbaf91ed447c779/packages/serve/src/index.ts#L27-L34

Reply:
Yes, throw an error for all unknown arguments:
```
node_modules/.bin/eslint --unknown
```

I was wrong about exit codes:
- 0 - all fine
- 1 - errors in the compilation
- 2 - problems with configurations/cli flags/etc (we even don't run build)